tiledb/sm/query/query.cc

```diff
@@ -163,6 +163,39 @@ Status Query::get_range(
   return reader_.get_range(dim_idx, range_idx, start, end, stride);
 }
 
+Status Query::get_range_var_size(
+    unsigned dim_idx,
+    uint64_t range_idx,
+    uint64_t* start_size,
```

Comment:
Check if `{start,end}_size` is var?

Reply:
I've added a check in the subarray class as this is where the other checks for for dim/range index out of bounds: https://github.com/TileDB-Inc/TileDB/pull/1608/files#diff-95d7879f8bd97183434a0c2ffcb3c74cR359 . Let me know if you'd rather have it up in the Query class.